changelog/2025-11-17T21_35_22+01_00_fixed_dumpvcd_trace_limit

```diff
@@ -0,0 +1 @@
+FIXED: Removed the limit on the number of traces in `dumpVCD`. Also removed unnecessary newlines after 1-bit signals. [#3082](https://github.com/clash-lang/clash-compiler/issues/3082)
```

Comment:
```suggestion
FIXED: Removed the limit on the number of traces in `dumpVCD`. Also removed unnecessary double newlines after 1-bit signals. [#3082](https://github.com/clash-lang/clash-compiler/issues/3082)
```

Reply:
A different way to phrase it would be

```suggestion
FIXED: Removed the limit on the number of traces in `dumpVCD`. Also removed unnecessary blank lines after 1-bit signals. [#3082](https://github.com/clash-lang/clash-compiler/issues/3082)
```

I think that's even more clear.